tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "watch": false,
     "baseUrl": "./packages",
     "paths": {
+      "@popmotion/easing": ["./easing/lib"],
```

Comment:
this was missing and preventing popmotion builds (which depends on this package)